src/Utils/DatetimeFactory/DatetimeFactoryInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace EDTF\Utils\DatetimeFactory;
+
+interface DatetimeFactoryInterface
+{
+    /**
+     * @param $year
+     * @param $month
+     * @param $day
+     * @param $hour
+     * @param $minute
+     * @param $second
+     * @param $tz
+     * @return mixed
+     *
+     * @throws DatetimeFactoryException
```

Comment:
I strongly prefer not repeating things that are already in the PHP, so would only include the throws tag